db/migrate/20231201194409_create_form_submissions.rb

```diff
@@ -0,0 +1,18 @@
+class CreateFormSubmissions < ActiveRecord::Migration[6.1]
+  def change
+    create_table :form_submissions do |t|
+      t.integer :form_type, default: 0, null: false
+      t.string :benefits_intake_uuid
+      t.string :submitted_claim_uuid
+      t.jsonb :form, default: {}
+      t.references :user_account, type: :string, column_name: :user_account_uuid
```

Comment:
From what I see in `schema.rb` it seems to be our convention to call the column name `:user_account_id` rather than `:user_account_uuid`. nbd either way but I just noticed it. Why can't we do this here:
```
t.references :user_account, foreign_key: true, type: :uuid
```

Reply:
this could work, my issue with calling it an `id` is that it's not a proper rails id, it's a string.  this comes with several behavioral differences (e.g. we can't index these models together because of the datatype mismatch).  Worth considering tho 🤔

Reply:
This was a FANTASTIC recommendation.  Not only does the updated syntax give us the naming convention `*_id` it also gives us cross-type indexing !!!!